sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_stream_eventhandler_with_toolset_async.py

```diff
@@ -69,32 +63,6 @@ async def on_done(self) -> None:
     async def on_unhandled_event(self, event_type: str, event_data: Any) -> None:
         print(f"Unhandled Event Type: {event_type}, Data: {event_data}")
 
-    async def _handle_submit_tool_outputs(self, run: "ThreadRun") -> None:
```

Comment:
Does all still work good?

Reply:
Yes.   I have tested all async samples, sync with stream with and without event handler and iteration.